### ISSUE TITLE
New tutorials: using a virtual environment

### ISSUE THREAD

**Issue by ddbeck** [pypa/packaging.python.org#202]
(For more information about this issue, please see #194).

We need a short section to follow the install/uninstall section (see #201) (i.e., as its own file). It should satisfy the following goals:
- show the reader to how to install virtualenv (if applicable, Pythons 3.3+ should come with pyenv, I think)
- show the reader how to create a virtual environment
- show the reader how to install something inside that environment (and verify that installing worked, as in the previous section)
- show the reader how to destroy the environment

In short, we want to equip the reader with the skills to install and use packages isolated from the system's site packages.

When opening pull requests for this issue, please submit to the `develop` branch. If you have questions that concern contributing more generally, please use issue #194. Otherwise, feel free to comment with questions or feedback. Thanks!

**Comment by amitsaha** [comment]
@ddbeck Can you please assign this one to me?

**Comment by ddbeck** [comment]
@amitsaha feel to free to start on this (for some reason GitHub won't let me literally assign the issue to you, but don't let my confusion with the UI slow you down)

**Comment by amitsaha** [comment]
@ddbeck  Great, as I commented in https://github.com/pypa/python-packaging-user-guide/pull/198 , can we decide on one third party package that we will use through out the docs? This can also be used in https://github.com/pypa/python-packaging-user-guide/pull/206 and elsewhere.

**Comment by ddbeck** [comment]
@amitsaha I liked @ncoghlan's suggestion of using the `pprintpp` package. But feel free to choose something that you're comfortable working with now—a draft is easier to edit than an idea. We don't have to commit to having the same sample throughout or we can work on achieving consistency through the editing process, so don't worry much about matching up with the other sections just yet.

**Comment by amitsaha** [comment]
@ddbeck  Just a note that I am slowly working on this at https://github.com/amitsaha/python-packaging-user-guide/blob/amitsaha_issue202/source/using-a-virtualenv.rst 

I have decided to use a separate package, because I felt this is something simpler.

**Comment by ddbeck** [comment]
Great, keep up the good work and be sure to open a pull request when you're ready to entertain feedback.

**Comment by amitsaha** [comment]
Thanks, i have opened a PR for preliminary feedback: https://github.com/pypa/python-packaging-user-guide/pull/211

**Comment by ddbeck** [comment]
The PR has been merged to develop, plus some style changes: https://packaging.python.org/en/develop/using-a-virtualenv/

Once more sections are written, I'll need to revisit this so that it appropriately signposts the following section (and lines up with the signposting in the previous section).

**Comment by tejaswini4996** [comment]
I will love to contribute #202
